### PR TITLE
resource/alicloud_alikafka_instance: Added the field resource_group_id

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_alikafka_instance_test.go [hunk 1]
```diff
@@ -319,42 +319,42 @@ func TestAccAliCloudAlikafkaInstance_convert(t *testing.T) {
...
-					"disk_type":       "1",
-					"disk_size":       "500",
-					"deploy_type":     "4",
-					"eip_max":         "3",
-					"io_max":          "20",
-					"vswitch_id":      "${data.alicloud_vswitches.default.ids.0}",
-					"paid_type":       "PostPaid",
-					"spec_type":       "normal",
-					"service_version": "2.2.0",
+					"name":              "${var.name}",
```
Comment (MrWolong) [inline alicloud/resource_alicloud_alikafka_instance_test.go:322]: === RUN   TestAccAliCloudAlikafkaInstance_basic
--- PASS: TestAccAliCloudAlikafkaInstance_basic (6126.60s)
PASS

=== RUN   TestAccAliCloudAlikafkaInstance_convert
--- PASS: TestAccAliCloudAlikafkaInstance_convert (225.02s)
PASS

=== RUN   TestAccAliCloudAlikafkaInstance_prepaid
--- PASS: TestAccAliCloudAlikafkaInstance_prepaid (217.93s)
PASS

=== RUN   TestAccAliCloudAlikafkaInstance_VpcId
--- PASS: TestAccAliCloudAlikafkaInstance_VpcId (1058.60s)
PASS

File: alicloud/resource_alicloud_alikafka_instance.go [hunk 2]
```diff
@@ -693,7 +713,47 @@ func resourceAliCloudAlikafkaInstanceUpdate(d *schema.ResourceData, meta interfa
 		d.SetPartial("config")
 	}
 
+	update = false
+	changeResourceGroupReq := map[string]interface{}{
+		"RegionId":   client.RegionId,
+		"ResourceId": d.Id(),
+	}
+
+	if d.HasChange("resource_group_id") {
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_alikafka_instance.go:722]: 这里可能会有问题，如果模板中把resource_group_id改成空字符串，会不会走到update中但没传参报错了

**Pull request merged**